Codeinwp/neve
# fix: aria label palette switch

### Summary
Added ARIA label for palette switch link

### Will affect the visual aspect of the product

NO

### Screenshots 

### Test instructions

1. Check that the palette switch on the front end has an ARIA label and it is being picked up by screen readers. 


Closes #3551.
